test/integration/index_documents_test.rb

```diff
@@ -0,0 +1,89 @@
+require "integration_test_helper"
+require 'govuk_message_queue_consumer'
+require 'govuk_message_queue_consumer/test_helpers'
+require './lib/index_documents'
+
+class IndexDocumentsTest < IntegrationTest
+  include IndexDocumentsTestHelpers
+
+  def setup
+    stub_elasticsearch_settings
+    stub_tagging_lookup
+    create_test_indexes
+    stub_calls_for_index_documents_test
+  end
+
+  def teardown
+    clean_test_indexes
+  end
+
+  def test_topics_get_indexed_from_publishing_api
+    m = GovukMessageQueueConsumer::MockMessage.new(payload_with_topics)
+    IndexDocuments.new.process(m)
+
+    assert m.acked?
```

Comment:
I think we need to be checking more than that the method acks the message.  Perhaps check that the appropriate payload was sent to elasticsearch?  Or that the topics from the payload were sent to elasticsearch, at least.